src/main/java/lotto/domain/BuyLotto.java

```diff
@@ -0,0 +1,76 @@
+package lotto.domain;
+
+import lotto.domain.constant.LottoRank;
+
+import java.util.List;
+import java.util.Objects;
+
+public class BuyLotto {
+
+    private final List<Lotto> lottos;
+
+    public BuyLotto(Integer... numbers) {
+        this(List.of(new Lotto(numbers)));
+    }
+
+    public BuyLotto(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public List<Lotto> getLottos() {
+        return lottos;
+    }
+
+    public LottoResult match(Lotto winningLotto, NumberElement bonusNumber) {
+        LottoResult lottoResult = new LottoResult();
+
+        for (Lotto buyLotto : lottos) {
+            lottoResult.putLottoResult(matchCount(buyLotto, winningLotto, bonusNumber));
```

Comment:
```suggestion
            lottoResult.putLottoResult(winningLotto.match(buyLotto)));
```
WinningLotto를 인자로 받는다면 위와 같이 접근하는 것이 가능하지 않을까?